leshan-client-core/src/main/java/org/eclipse/leshan/client/resource/DummyInstanceEnabler.java

```diff
@@ -56,12 +56,23 @@ public ReadResponse read(ServerIdentity identity, int resourceid) {
         return super.read(identity, resourceid);
     }
 
+    public ReadResponse read(ServerIdentity identity, int resourceid, int resourceInstance) {
```

Comment:
missing @Override but I can add it later if you want

Reply:
Done